src/components/standalone/monitoring/NetifyInformaticsContent.vue

```diff
@@ -0,0 +1,188 @@
+<!--
+  Copyright (C) 2024 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { ref, onMounted, watch } from 'vue'
+import type { Ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+import FormLayout from '@/components/standalone/FormLayout.vue'
+import { useNotificationsStore } from '@/stores/notifications'
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { NeButton } from '@nethesis/vue-components'
+import {
+  NeSkeleton,
+  NeFormItemLabel,
+  NeTextInput,
+  getAxiosErrorMessage,
+  NeInlineNotification,
+  NeBadge
+} from '@nethesis/vue-components'
+import { NeToggle } from '@nethesis/vue-components'
+import { ubusCall } from '@/lib/standalone/ubus'
+import { AxiosError } from 'axios'
+
+const notificationsStore = useNotificationsStore()
+const { t } = useI18n()
+
+interface Form {
+  status: boolean
+  uuid: string
+}
+
+const form = ref<Form>({
+  status: false,
+  uuid: ''
+})
+
+interface ErrorObject {
+  notificationTitle: string
+  notificationDescription: string
+  uuid: string
+}
+const objError: ErrorObject = {
+  notificationTitle: '',
+  notificationDescription: '',
+  uuid: ''
+}
```

Comment:
While I don't like the approach of multiple error handling like this, I can allow it, but it completely loses effectiveness when there are multiple variables for each error, is redundant (and don't assign them like that, while TS allows it, it's really a nasty approach)